src/services/game/generateQuestions.js

```diff
@@ -23,7 +23,7 @@ const getRangeOfIds = async (gameMode) => {
 const generateAnswerNumbers = (arrOfNumbers) => {
   const minIndex = 0;
   const answerNumbers = [];
-  for (let i = 0; answerNumbers.length <= 4; i++) {
+  for (let i = 0; answerNumbers.length <= 3; i++) {
```

Comment:
why 3?

@axseinga

Reply:
<= 3 is correct, as the loop goes from zero so it should return 4 numbers. I had it as 3 in this line of code before but Magda showed me that there is a bug and it only returns 3 numbers/answers (with her part of code, . I assumed that this was the bug, I remember I tested it with Magda's code and it was 4 numbers/answers but now when I tested the code it returns 5 numbers so Krzysztof is right about changing it back to 3. No idea, maybe the bug was somewhere else and was caused by object ids as you know we had lots of issues because of missing numbers etc and I was fixing it on the way:P anyway 3 is correct :)